packages/quantic/force-app/main/default/lwc/quanticRecommendationInterface/quanticRecommendationInterface.js

```diff
@@ -59,6 +59,8 @@ export default class QuanticRecommendationInterface extends LightningElement {
   hasRendered = false;
   /** @type {boolean} */
   ariaLiveEventsBound = false;
+  /** @type {string} */
+  analyticsOriginContext = 'recommendations';
```

Comment:
@operecoveo @DavAtallah 
Not exactly sure how to validate this, but this is where we decide what to send to analytics under the `originContext` value

Reply:
Do we merge this @erocheleau ?

Reply:
`quantic_recommendation`